tests/actor/test_data_sourcing.py

```diff
@@ -25,22 +24,18 @@ async def test_data_sourcing_actor() -> None:
     server = mock_api.MockGrpcServer(servicer, port=57899)
     await server.start()
 
-    servicer.add_component(1, components_pb.ComponentCategory.COMPONENT_CATEGORY_GRID)
-    servicer.add_component(4, components_pb.ComponentCategory.COMPONENT_CATEGORY_METER)
-    servicer.add_component(7, components_pb.ComponentCategory.COMPONENT_CATEGORY_METER)
-    servicer.add_component(
-        8, components_pb.ComponentCategory.COMPONENT_CATEGORY_INVERTER
-    )
-    servicer.add_component(
-        9, components_pb.ComponentCategory.COMPONENT_CATEGORY_BATTERY
-    )
+    servicer.add_component(1, ComponentCategory.GRID)
+    servicer.add_component(4, ComponentCategory.METER)
+    servicer.add_component(7, ComponentCategory.METER)
+    servicer.add_component(8, ComponentCategory.INVERTER)
+    servicer.add_component(9, ComponentCategory.BATTERY)
 
     servicer.add_connection(1, 4)
     servicer.add_connection(1, 7)
     servicer.add_connection(7, 8)
     servicer.add_connection(8, 9)
 
-    await connection_manager.initialize("[::1]", 57899)
+    await connection_manager.initialize("grpc://[::1]:57899")
```

Comment:
looks like `Use new microgrid API wrapped types` is not the right commit for this change

Reply:
Moved to the commit introducing the `server_url` :+1: